xml/http_wwwauth.xml

```diff
@@ -444,6 +444,12 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:f5:big-ip_local_traffic_manager:-"/>
   </fingerprint>
 
+  <fingerprint pattern="Prowlarr">
```

Comment:
This regex will match anything that contains `Prowlarr`. It should be constrained to the HTTP WWW-Authenticate headers as the other fingerprint regex in this file.